### DISCUSSION TITLE
Notification issue

### DISCUSSION THREAD

**Discussion by Shahriyar13** [community/community#122113] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

In my profile in notifications page, there is an invisible notification which I can not see, delete, mark as read & ... 
I checked every thing, every filter, without filters, press every shortcut buttons, ...

![Screenshot 2024-05-03 093958](https://github.com/community/community/assets/33436153/d0eafa8f-17bd-4801-96b3-80fc7812a743)

**Comment by deandreamatias** [comment]
I also have this issue
Is associated to a repo that do not exists
![imagen](https://github.com/community/community/assets/21011641/6c1cc4f7-1505-440d-9db9-ea0444cef0dd)

**Comment by Naddiseo** [comment]
This seems to happen if github themselves notice one of those spam repositories or users then deletes the offending repo or issue without clearing everyone's notifications. I'm also suffering from this.

**Reply by Naddiseo** [reply]
Looks liek the answer here works: https://github.com/orgs/community/discussions/32380#discussioncomment-3785542

**Comment by NoelTautges** [comment]
I also have this issue, but [the workaround linked above](https://github.com/orgs/community/discussions/122113#discussioncomment-9572926) doesn't work for me.

**Comment by 0x5bfa** [comment]
I also have the issue.

![image](https://github.com/community/community/assets/62196528/faa557e9-b94c-4d4c-adb6-5193f388bcd8)

**Reply by 0x5bfa** [reply]
This looks like fixed if you add items back from Done and remove back.

**Reply by deandreamatias** [reply]
I made this and isn't works

**Reply by 0x5bfa** [reply]
Forgot this, seems to have been fixed.

**Comment by wangeguo** [comment]
I also have the issue.

**Comment by deandreamatias** [comment]
Seems to have been resolved! Take a look

**Comment by solairen** [comment]
I also had it, did nothing and it disappeared after a month or so 😄
